hkcert2020-calm-down/server_files/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:latest
```

Comment:
Please use a specific version of Ubuntu so that the challenge continues to build in the future - maybe `ubuntu:20.04` ?